software/webhook-tester.yml

```diff
@@ -0,0 +1,11 @@
+name: "WebHook Tester"
+source_code_url: "https://github.com/tarampampam/webhook-tester"
```

Comment:
I believe website url is required.
`suggestion
source_code_url: "https://github.com/tarampampam/webhook-tester"
website_url: "https://github.com/tarampampam/webhook-tester"
```

Reply:
Done, thank you!